src/Spinner.tsx

```diff
@@ -11,11 +11,12 @@ const sizeMap = {
 }
 
 export interface SpinnerInternalProps {
+  /** Sets the width and height of the spinner. */
   size?: keyof typeof sizeMap
 }
 
-function Spinner({size: sizeKey, ...props}: SpinnerInternalProps) {
-  const size = (sizeKey && sizeMap[sizeKey]) ?? sizeMap.medium
+function Spinner({size: sizeKey = 'medium', ...props}: SpinnerInternalProps) {
```

Comment:
Hmm, looks like `react-docgen-typescript` doesn't recognize `medium` as the default `size` value